packages/expo-web-browser/ios/EXWebBrowser/EXWebBrowser.m

```diff
@@ -205,4 +242,23 @@ - (void)setModuleRegistry:(UMModuleRegistry *)moduleRegistry
   _moduleRegistry = moduleRegistry;
 }
 
+- (UIColor *)convertHexColorString:(NSString *)stringToConvert {
```

Comment:
```suggestion
+ (UIColor *)convertHexColorString:(NSString *)stringToConvert {
```

`+` makes a method static, we're using it as `[EXWebBrowser color…]` in the implementation